test/groovy/MtaBuildTest.groovy

```diff
@@ -73,7 +73,7 @@ public class MtaBuildTest extends BasePiperTest {
 
 
     @Test
-    void mtarFilePathFromCommonPipelineEnviromentTest() {
+    void marFilePathFromCommonPipelineEnvironmentTest() {
```

Comment:
Typo, I would like to suggest to fix this before merging

Reply:
I applied the fix suggested by intellij, so is there another typo?

Reply:
Oh, I spotted it. IntelliJ screwed up mta.